File: 프로그래머스/unrated/181838. 날짜 비교하기/날짜 비교하기.py
def solution(date1, date2):
    date1Sum = str(date1[0])+str(date1[1])+str(date1[2]);
    date2Sum = str(date2[0])+str(date2[1])+str(date2[2]);
    return 1 if int(date1Sum) - int(date2Sum) < 0 else 0

File: 프로그래머스/unrated/181838. 날짜 비교하기/날짜 비교하기.js
function solution(date1, date2) {
//     const [y1, m1, d1] = date1;
//     const [y2, m2, d2] = date2;
    
//     return +(y1 < y2 || (y1 === y2 && m1 < m2) || (y1 === y2 && m1 === m2 && d1 < d2))
    
    return +(date1.join('') - date2.join('') < 0);
    
    // return +(new Date(date1) < new Date(date2))
}